source/index.html.md.erb

```diff
@@ -24,57 +22,57 @@ Use the EMCS API to:
 
 ## API status
 
-This version of the EMCS API is in [the beta phase](https://www.gov.uk/service-manual/agile-delivery/how-the-beta-phase-works).
+This version of the EMCS API:
+
+- is in [the beta phase](https://www.gov.uk/service-manual/agile-delivery/how-the-beta-phase-works)
+- is currently ready for testing
+- will not be ready for use in production until EMCS FS 4.0 goes live. In the meantime, you should continue to use the SOAP EMCS API.
 
 ## Getting started
 
 ### Using EMCS API for the first time
 
-If you are new to the EMCS API, you should:
+If you are new to the EMCS API, you should follow these steps:
 
-1. Review all of this document before reviewing other documents for EMCS API
-2. Follow all of the steps in [First-time EMCS API users](/#first-time-emcs-api-users).
-3. Review the [testing guide]()
-4. Review the [technical interface specification]()
-5. Review [Making API requests](/#making-api-requests)
+- Read all of this document before reviewing other documents for EMCS API
+- Follow all of the steps in [First-time Developer Hub users](/#first-time-developer-hub-users).
+- Read the [EMCS API testing guide](/guides/emcs-api-testing-guide/)
+- Read the [EMCS API technical interface specification]()
+- Read [Making API requests](/#making-api-requests)
 
 ### Migrating from an earlier version
 
-If you are migrating from the old SOAP EMCS API, you should:
-
-1. Review the [migrating from SOAP API page](/documentation/migration-guide.html) before reviewing other documents for EMCS API
-2. Review the [testing guide]()
-3. Review the [technical interface specification]()
-4. Review [Making API requests](/#making-api-requests)
-
-### Trader data
+If you are migrating from the old SOAP EMCS API, you should read:
 
-- Information you'll need from traders if you work for a software house.
+- the [migrating from SOAP API page](/documentation/migration-guide.html) before reviewing other documents for EMCS API
+- the [EMCS API testing guide](/guides/emcs-api-testing-guide/)
+- the [EMCS API technical interface specification]()
+- [Making API requests](/#making-api-requests)
 
-### First-time EMCS API Users
+### First-time Developer Hub Users
 
-If you have not previously used the EMCS API, you should:
+If you have not previously used the Developer Hub, you should:
 
-1. Ensure that you have an HMRC [developer account](https://developer.service.hmrc.gov.uk/developer/login) - if you do not have one, you must [register for an account](https://developer.service.hmrc.gov.uk/developer/registration), activate it by email, and sign in.
+1. [Sign up for an HMRC developer account](https://developer.service.hmrc.gov.uk/developer/registration), activate it by email, and [sign in](https://developer.service.hmrc.gov.uk/developer/login)).
 2. Add your subscription to this API to your application software.
```

Comment:
this step applies to you if you have used the Developer Hub but not the EMCS API, so maybe needs to be in a separate section?

Reply:
Retitled the section, moved some text around.